src/odemis/gui/cont/fastem_acq.py

```diff
@@ -512,14 +512,23 @@ def on_acquisition(self, evt):
         # Acquire ROAs for all projects
         fs = {}
         pre_calibrations = [Calibrations.OPTICAL_AUTOFOCUS, Calibrations.IMAGE_TRANSLATION_PREALIGN]
+        pre_calib_plus = [Calibrations.OPTICAL_AUTOFOCUS,
+                          Calibrations.AUTOSTIGMATION,
+                          Calibrations.IMAGE_TRANSLATION_PREALIGN]
+
+        # Read the frequency with which to run autostigmation from the acquisition configuration
+        acqui_conf = conf.get_acqui_conf()
+        autostig_freq = math.inf if acqui_conf.autostig_freq == 0 else acqui_conf.autostig_freq
+
         for p in self._tab_data_model.projects.value:
-            for roa in p.roas.value:
+            for idx, roa in enumerate(p.roas.value):
+                pre_calib = pre_calib_plus if idx % autostig_freq == 0 else pre_calibrations
```

Comment:
Based on https://github.com/delmic/odemis/pull/2624/files#r1464691372, I have a question. If the `autostig_freq = 5`, do we only want to use `pre_calib_plus` for ROAs with index `0, 5, 10, ...`?

Reply:
yes